src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1913,9 +1917,15 @@ private static IEnumerable<string> ParseWarnings(string value)
             string[] values = value.Split(new char[] { ',', ';', ' ' }, StringSplitOptions.RemoveEmptyEntries);
             foreach (string id in values)
             {
-                ushort number;
-                if (ushort.TryParse(id, NumberStyles.Integer, CultureInfo.InvariantCulture, out number) &&
-                    ErrorFacts.IsWarning((ErrorCode)number))
+                if (id.ToLowerInvariant() == "nullable")
```

Comment:
>if (id.ToLowerInvariant() == "nullable") [](start = 16, length = 40)

Probably should use `if (string.Equals(accessibility, "nullable", StringComparison.OrdinalIgnoreCase)) ...` as the rest of this file does

Reply:
Yes, that should avoid allocating an extra string, and Invariant is the wrong comparer for strings.